StepperEngine/src/Flow/FlowRunHistory.java

```diff
@@ -0,0 +1,89 @@
+package Flow;
+
+import DataTypes.DataType;
+import Steps.Step;
+
+public class FlowRunHistory {
+    private String flowId;
+    private String flowName;
+    private Flow.Status status;
+    private double runTime;
+    private String timeStamp;
+
+    private String freeInputsHistory;
+    private String outputsHistory;
+    private String stepsHistory;
```

Comment:
also, what about get() methods for those Strings?

Reply:
they will be added on next pr